### PR TITLE
Allow multiple config sets

### PR DESCRIPTION
**Pull request by karelhala**
### Description

Since we want to change the url everytime consumer changes the active workspace we have to re-set the utils config on each update.

### CODE REVIEW

File: packages/lib-utils/src/app/AppInitSDK.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import type { PluginStore } from '@openshift/dynamic-plugin-sdk';
 import * as React from 'react';
 import { Provider } from 'react-redux';
 import type { UtilsConfig } from '../config';
-import { isUtilsConfigSet, setUtilsConfig } from '../config';
```
[the remark this answers is not in the document]
Comment (karelhala) [reply packages/lib-utils/src/app/AppInitSDK.tsx:7]: It's exposed trough index.js I'd rather add `@deprecated` flag to it

[the remark this answers is not in the document]
Comment (vojtechszocs) [reply packages/lib-utils/src/app/AppInitSDK.tsx:7]: This is also why I'm generally against exporting everything by default from a library - it tends to blur the boundaries between what's public and what's internal.

[the remark this answers is not in the document]
Comment (karelhala) [reply packages/lib-utils/src/app/AppInitSDK.tsx:7]: Unfortunately it's used in hac-core - https://github.com/openshift/hac-core/blob/main/frontend/src/Routes/testK8s/DetermineNamespace.tsx#L14 it's used only in testing place, so I don't think it's that necessary. Personally, I'd add deperecated, however as this basically has to happen before even using any API request I am fine with removing it as well.

**Comment by karelhala** [comment]
Since we won't support KCP we don't need multiple config sets. Closing.

**Pull request closed without merging**